app/src/util/tools.ts

```diff
@@ -66,6 +66,17 @@ export const formatDate = (date: Date, utcAdd = true): string => {
     .tz('UTC')
     .format(`YYYY-MM-DD - HH:mm${utcAdd ? ' [UTC]' : ''}`)
 }
+export const formatLockDate = (date: number) => {
+  return moment(date)
+    .tz('UTC')
+    .format(`MMMM Do, YYYY - HH:mm UTC`)
+}
+export const daysUntil = (timestamp: number): number => {
+  const current = moment()
+  const day = moment.unix(timestamp)
+
+  return Math.ceil(moment.duration(day.diff(current)).asDays())
+}
```

Comment:
Do we have tests for these?

Reply:
So the issue with that is daysUntil uses current time to calculate and I couldn't create tests reliably with that? Do you have some suggestion maybe? But will create tests for formatLockDate!

Reply:
Maybe this might help: https://dev.to/jbranchaud/test-timing-based-js-functions-with-jest-5be

Reply:
Thank you for the feedback on keybase! Fixed it!